### PR TITLE
Allow for compatibility for pandas>=0.25.3

### LINKED ISSUE #41: Pandas 1.0 Options API Tweaks
**Bug Description**
[`df_to_html`](https://github.com/qiime2/q2templates/blob/6d2e26c56a618ed290e89c1983fdd1a072e7a903/q2templates/util.py#L45) sets the `display.max_colwidth` parameter to -1 to disable truncating the cell values. This parameter value (`-1`) is no longer supported, and should be replaced with `None` to opt into the same behavior

**Steps to reproduce the behavior**
1. Install pandas 1.0: `conda install -c conda-forge -c bioconda pandas=1.0`
2. 
```python
import pandas as pd
from q2templates import df_to_html

df = pd.DataFrame([])
df_to_html(df)
```
3. Observe the following error: `Value must be a nonnegative integer or None`

**Expected behavior**
No error.

**References**
1. [Busywork](https://busywork.qiime2.org/teams/main/pipelines/master-distribution/jobs/unit-test-q2-demux/builds/65).

### PR DESCRIPTION
**Pull request by andrewsanchez**
closes #41 

I tried to be cute with `max_colwidth = None if float(pd.__version__) >= 1.0 else -1` but...
```python
>>> float("0.25.3")
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
ValueError: could not convert string to float: '0.25.3'
```

:cry:

### CODE REVIEW

File: q2templates/util.py [hunk 1]
```diff
@@ -42,8 +42,13 @@ def df_to_html(df, border=0, classes=('table', 'table-striped', 'table-hover'),
     .. [2] https://github.com/pandas-dev/pandas/issues/1852
 
     """
-    with pd.option_context('display.max_colwidth', -1):
-        return df.to_html(border=border, classes=classes, **kwargs)
+    try:
```
Comment (thermokarst) [inline q2templates/util.py]: If we [only support pandas 1.0](https://github.com/qiime2/q2templates/issues/41#issuecomment-582037132) we can drop this branch of code - does that work for you?

Comment (andrewsanchez) [reply q2templates/util.py]: Yeah, I agree.  Seems like that would be preferable.

**Comment by andrewsanchez** [comment]
It looks like tests fail in the Travis environment because pandas 0.25.3 is installed as per `qiime2-latest-py36-linux-conda.yml`, rather than the pandas version pinned in the recipe and the version that will be tested against later by busywork.

Isn't configuring an qiime2 environment unnecessary for building/testing this repo?  Seems like a waste of time and resources given the limited dependencies in `meta.yaml`.  Should we modify `.travis.yml` to match up with `meta.yaml` or am I missing something?

**Comment by thermokarst** [comment]
> It looks like tests fail in the Travis environment because pandas 0.25.3 is installed as per `qiime2-latest-py36-linux-conda.yml`, rather than the pandas version pinned in the recipe and the version that will be tested against later by busywork.

No worries, this is expected - there is a bit of a chicken-and-egg problem w.r.t. dependencies and our default travis config. You are totally right that travis isn't using the pandas version pinned in the recipe - because travis will only use the latest staging environment created by busywork (🐔, meet 🥚).

> Isn't configuring an qiime2 environment unnecessary for building/testing this repo?

Totally, which is why I'm going to let busywork and travis sort this all out. Your changeset looks good to me, so I will merge this (even though travis is failing), and see what happens on busywork's side of things.

Sorry for the confusing workflow here!

**Pull request merged**